content/shared/influxdb3-plugins/plugins-library/official/basic-transformation.md

```diff
@@ -197,7 +200,47 @@ influxdb3 create trigger \
   --trigger-arguments 'measurement=products,window=7d,target_measurement=products_updated,values_transformations=status:"status_replace",custom_replacements=status_replace:"In Stock=available.Out of Stock=unavailable"' \
   status_updater
 ```
+## Using TOML Configuration Files
```

Comment:
We should replace the previous `../../README.md` link with a link fragment pointing here.